modules/enabling-additional-enabled-capabilities.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="setting_additional_enabled_capabilities_{context}"]
+= Enabling the cluster capabilities by setting additional enabled capabilities
+
+As a cluster administrator, you can enable the cluster capabilities by setting `additionalEnabledCapabilities`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
```

Comment:
@darshan-nagaraj We are attempting to not refer customers to use jq:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output which says:

Do not use jq in commands (unless it is truly required), because this requires users to install the jq tool. Oftentimes, the same or similar result can be accomplished using jsonpath for oc commands.

What we should instead be doing is using the jsonpath option for oc commands. the guidelines there give a few ways on how to change it. 

Please, can you check with your SME to find out if the same thing can be accomplished with jsonpath instead?
And, if not, I would still recommend removing the link to download jq.

Reply:
I was skeptical of the jq but I was not sure if there is a better alternative to it. So support the idea of replacing jq with jsonpath.

Reply:
@GroceryBoyJr @LalatenduMohanty I have pushed the changes.